packages/editable-html-tm/src/index.jsx

```diff
@@ -38,7 +38,7 @@ function dataImageHandler(blobInfo, success, failure, progress) {
   }, 500);
 }
 
-function EditorHtml({ classes, markup, onChange, onDone, height, width, outputFormat, onBlur, onFocus }) {
+function EditorHtml({ classes, markup, onChange, onDone, height, width, outputFormat, onBlur, onFocus, fixedToolbarContainer, tinyMCEApiKey }) {
```

Comment:
Do we use fixedToolbarContainer or we have any plans for it soon?
Moving tinyMCE key the at the app level is great, I added it here for dev purpose only as you work on pie-elements in that repo and you still have to pass some key to not show that warning, but its place should be at the app level indeed